airbyte-bootloader/src/main/java/io/airbyte/bootloader/BootloaderApp.java

```diff
@@ -104,18 +143,25 @@ public void load() throws Exception {
     LOGGER.info("Setting up config database and default workspace..");
 
     try (
-        final Database configDatabase =
-            new ConfigsDatabaseInstance(configs.getConfigDatabaseUser(), configs.getConfigDatabasePassword(), configs.getConfigDatabaseUrl())
-                .getAndInitialize();
-        final Database jobDatabase =
-            new JobsDatabaseInstance(configs.getDatabaseUser(), configs.getDatabasePassword(), configs.getDatabaseUrl()).getAndInitialize()) {
+        final Database configDatabase = new ConfigsDatabaseInstance(configsDslContext).getAndInitialize();
+        final Database jobDatabase = new JobsDatabaseInstance(jobsDslContext).getAndInitialize()) {
       LOGGER.info("Created initial jobs and configs database...");
 
       final JobPersistence jobPersistence = new DefaultJobPersistence(jobDatabase);
       final AirbyteVersion currAirbyteVersion = configs.getAirbyteVersion();
       assertNonBreakingMigration(jobPersistence, currAirbyteVersion);
 
-      runFlywayMigration(configs, configDatabase, jobDatabase);
+      // TODO Will be converted to an injected singleton during DI migration
+      final Flyway configsFlyway = FlywayFactory.create(configsDataSource, BootloaderApp.class.getSimpleName(), ConfigsDatabaseMigrator.DB_IDENTIFIER,
+          ConfigsDatabaseMigrator.MIGRATION_FILE_LOCATION);
+      final Flyway jobsFlyway = FlywayFactory.create(jobsDataSource, BootloaderApp.class.getSimpleName(), JobsDatabaseMigrator.DB_IDENTIFIER,
+          JobsDatabaseMigrator.MIGRATION_FILE_LOCATION);
```

Comment:
Why aren't we passing these Flyway objects into the constructor now, like we are doing with the DSLContext and Datasource?

Reply:
@lmossman Trying to limit the code changes.  Ultimately, these will be injected.  Additionally, they don't require closing on shutdown, so I took the path of least resistence.